### PR TITLE
Fixed choosing orientation for some previous INVALID conditions with greedy algorithm.

### PR DESCRIPTION
**Pull request by blowekamp**
Addresses SimpleITK/SimpleITK#1375

### CODE REVIEW

**Comment by blowekamp** [comment]
The same issue exits in itk::OrientImageFilter.

File: test/itkDICOMOrientImageFilterGTest.cxx [hunk 1]
```diff
@@ -450,8 +450,15 @@ TEST(DICOMOrientation, DirectionCosinesToOrientation)
   d(1, 1) = 1;
```
Comment (dzenanz) [inline test/itkDICOMOrientImageFilterGTest.cxx:450]: in commit message and -> an

File: src/itkDICOMOrientation.cxx [hunk 2]
```diff
@@ -170,27 +170,50 @@ DICOMOrientation::DirectionCosinesToOrientation(const DirectionType & dir)
   // but it is DIFFERENT in the meaning of direction in terms of sign-ness.
```
Comment (dzenanz) [inline src/itkDICOMOrientation.cxx:170]: in commit message closes -> closest

Comment (dzenanz) [reply src/itkDICOMOrientation.cxx:170]: dominate -> dominant

File: src/itkDICOMOrientation.cxx [hunk 3]
```diff
@@ -170,27 +170,50 @@ DICOMOrientation::DirectionCosinesToOrientation(const DirectionType & dir)
...
+      {
+      ++it;
+      }
+    }
 
-    switch (dominant_axis)
+    switch (max_c)
     {
       case 0: {
         // When the dominate axis sign is positive, assign the coordinate for the direction we are increasing towards.
```
Comment (dzenanz) [inline src/itkDICOMOrientation.cxx]: Is `dominate` an adjective? If not, change it to `dominant` (same as in commit message).

**Comment by dzenanz** [comment]
> The same issue exits in itk::OrientImageFilter.

Can you update it in the same way?

**Review by zivy — APPROVED** [review]

**Review by dzenanz — APPROVED** [review]
In second commit's title `closes` -> `closest`.

**Comment by blowekamp** [comment]
Thanks for the reviews :)

**Pull request merged**